src/hooks/usePaymentInitiator.ts

```
@@ -0,0 +1,925 @@
+                (isXChain || diffTokens) && calculatedSlippage
+                    ? {
+                          expected: Number(calculatedSlippage) * EXPECTED_SLIPPAGE_MULTIPLIER,
+                          max: Number(calculatedSlippage),
+                      }
+                    : undefined
+
+            const totalMax =
+                Number(estimatedFromValue) * selectedTokenData!.price + networkFee.max + (slippage?.max || 0)
+
+            const formatNumberSafely = (num: number) => {
+                if (isNaN(num) || !isFinite(num)) return '0.00'
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)